data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/drains/drains_multiply.cfg

```diff
@@ -0,0 +1,33 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [drains]value=
+##
+# Actions:
+# Give the leaders a drains ability with both the value and multiply attributes
+# Have the side 1 leader attack the side 2 leader with both its weapons
+##
+# Expected end state:
+# The both leaders have 2 weapons each of which now drains 8 hp (2*4)
+# The leader of side 1 heals 6 hp total since it has full hp when its first strike is made
```

Comment:
either this or previous line is wrong

Reply:
Yeah, I copied and updated a previous set of tests and I missed updating that.